### PR TITLE
Feat/recipe testing suite

### PR DESCRIPTION
**Pull request by WinstonCalhoun**
#### What's this PR do?
This PR is adding the Recipe class and testing as well as the Ingredient class and testing.

#### Where should the reviewer start?
Start with the Ingredient class and testing.

#### What are the relevant tickets?
Relevant ticket#7 for this PR has been resolved.

### CODE REVIEW

File: src/data/ingredientTestData.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  {
+    "id": 2050,
+    "name": "vanilla",
+    "estimatedCostInCents": 926
+  }
+]
+module.exports = ingredientTestData;
+
+
+//REMOVE THE CURLIES
```
Comment (sophielabelle) [inline src/data/ingredientTestData.js:61]: We need to remove this comment

**Review by sdmisra — APPROVED** [review]
This looks great - some refactoring to do later such as comment lines at the bottom of ingredientTestData.js.

**Review by sophielabelle — CHANGES_REQUESTED** [review]
Looks good aside from removing the comment on line 61 of src/data/ingredientTestData.js.
Once that is resolved we are ready to move on to finishing the user class and test suite.

File: src/data/ingredientTestData.js (hunk 1 above)
Comment (sdmisra) [reply src/data/ingredientTestData.js:61]: I agree

**Review by sophielabelle — APPROVED** [review]
Looks great! Now for users

**Pull request merged**